Makefile

```diff
@@ -28,7 +28,7 @@ tools: execprog mutate prog2c stress repro upgrade db
 
 # executor uses stacks of limited size, so no jumbo frames.
 executor:
-	$(CC) -o ./bin/syz-executor executor/executor.cc -pthread -Wall -Wframe-larger-than=8192 -Wparentheses -Werror -O1 -g $(STATIC_FLAG) $(CFLAGS)
+	$(CC) -o ./bin/syz-executor executor/executor.cc -lstdc++ -pthread -Wall -Wframe-larger-than=8192 -Wparentheses -Werror -O1 -g $(STATIC_FLAG) $(CFLAGS)
```

Comment:
I feel this will cause problems on Windows/Android/fuchsia and who knows what. We tried to avoid libc++ all that time. Also we avoid any unnecessary/hidden syscalls in executor, namely we want to control all syscalls done by executor and don't use any dynamic memory. Heap can be corrupted. Or vector-caused mmaps can disturb execution.
Please refrain from using std::vector.